### DISCUSSION TITLE
How to get Special Characters (&)?

### DISCUSSION THREAD

**Discussion by gagestinson** [oetiker/acronym#57] [Q&A]
I am very new to LaTeX. I am trying to write up a document for work. I want to make the acronym "P&C" but of course that does not work. I am needing some help with this. Redefining the commands, anything. Any shortcut to get this to work and still have the links and all that work. I know the issue comes from `csname` but I do not know how to alter that. Any help is much appreciated.

**Comment by Raphy312** [comment]
Would this work?

\begin{acronym}[ACRONYM]
    \acro{pc}[P\\&C]{ABC}
\end{acronym}

Hello \ac{pc} defg!
